python/example_code/iot/thing_performance.py

```diff
@@ -147,7 +147,7 @@ def run(self):
 
     # Configures the MQTT shadow client for this thing.
     def configureMQTTClient(self):
-        mqttClient = AWSIoTMQTTShadowClient(self.thingName)
+        mqttClient = AWSIoTPyMQTT.AWSIoTMQTTShadowClient(self.thingName)
```

Comment:
Hey, @mperusk-aws. These changes don't look functionally different to me. Is there some context I'm missing here?

Reply:
The SDE gave me that line. We were calling on the `AWSIoTMQTTShadowClient` previously. And now it's `AWSIoTPyMQTT` per the SDK. I was trying to match the syntax they see in the SDK: https://s3.amazonaws.com/aws-iot-device-sdk-python-docs/sphinx/html/generated/AWSIoTPythonSDK.MQTTLib.html.

Reply:
I see, I think there's an error here then. This is still using the ShadowClient. I think this line should be changed. I'll also comment on the import.
```suggestion
        mqttClient = AWSIoTMQTTClient(self.thingName)
```